portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,19 +31,18 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-   // Converts message to JSON string
+    // Converts message to JSON string
     String message = messageGson(messages());
     response.setContentType("application/json;");
     response.getWriter().println(message);
   }
-    // Will have messages that are presented on the screen
 
+// Will have messages that are presented on the screen
 private ArrayList<String> messages() {
-    String realMes = "";
     ArrayList<String> message = new ArrayList<String>();
```

Comment:
messageList, maybe, since it's a list of messages?

Reply:
solved